openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/SCSoftmxTestUtil.java

```diff
@@ -100,16 +100,16 @@ else if (task.equals(COMPARE_AOT_JIT_DATA_SPACE)) {
 			scanner1.close();
 			scanner2.close();
 
-			if (aotValue2 > aotValue1) {
+			if (aotValue2 >= aotValue1) {
 				System.out.println("New AOT data was written to the AOT space after softmx increase");
```

Comment:
The messages are a bit misleading in the case where the values are equal.

Reply:
ah.. missed that. @hangshao0 - what would you like me to put here as message since we are accepting the case of NO new byte written as well?

Reply:
For the equal case, you can use the message in the current else block "No new AOT data was written to the AOT space after softmx increase". A new message should be used for StfException.

Reply:
OK. Broke down the check as suggested. Please take a look.